src/Enghouse/readme.md

```diff
@@ -0,0 +1,100 @@
+4. By default the name presented to the contact-centre is the Watson user_id - this display name can be changed in the Flow by setting variables (case of names ignored) as per [Dialog example](./VG_TravelDemo-dialog.json), i.e. 
+- firstName - if this is set without lastName, this is the display name
+- lastName - if the firstName is also set, the display name is firstName and lastName (with a space between)
+- userId - if the firstName is not set this may set the display name
+
+
+## Example
+
+If the CustomChat "widget" URL was:
+
+`https://{Enghouse CCaaS FQDN}/SocialConnectorHelperAPI/api/customchat/getWidgetConfiguration?WidgetId=<*widgetID*>`
+
+...you would set this to the variable *CCURL* in the Dialog, prior to any likelihood of needing to speak to a Live Agent.
+
+
+If the Assistant URL provided to you by Enghouse Interactive was:
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat`
+
+...you would access your Assistant dialog pass your Assistants' parameters to it (using default EI **CCaaS** Watson page in this example):
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat/indexWatson.html?integrationID=<*integration-ID*>&region=<*region*>&serviceInstanceID=<*service-ID*>`
+
+...or if you preferred not to change an existing Dialog, the "widget" URL may be added as a URL-encoded parameter instead of being set in the dialog:
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat/indexWatson.html?integrationID=<*integration-ID*>&region=<*region*>&serviceInstanceID=<*service-ID*>&ccURL=<*widget-URL*>`
+
+
+## URL options
+
+The complete list of parameters that may be passed to the standard EI **CCaaS** Watson page URL are:
+
+- *integrationID* - Watson Assistant Integration ID
+- *region* - Watson Assistant region
+- *serviceInstanceID* - Watson Assistant Instance ID
+- *subscriptionID* - Watson Assistant Subscription ID
+- *ccName* - the display name of the contact centre, displayed prior to agent answering chat (default is "Contact Centre")
+- *ccURL* - the URL-encoded "widget" URL. This may also be set (unencoded) via the Dialog as the *CCURL* variable (overrides URL if both set). 
+- *agentIcon* - the URL-encoded URL of the agent icon (default is an Enghouse Interactive swirl logo).  This may also be set (unencoded) in the Dialog as the *AgentIcon* variable (overrides URL if both set).
+
+In the likely event of using your own HTML page, the last 3 are parameters you may pass to the `EnghouseCCaaSWatsonAssistantSD` ServiceDesk obtained via your preferred means.
+
+## Demo
+![IBMWatson](https://user-images.githubusercontent.com/30227436/203360082-bd2d1968-0d35-47e3-8039-0acd852bbf67.gif)
```

Comment:
It was really great that you added this but since no good deed goes unpunished, I'm going to suggest that you read do this animation/video. Since you made this we released a major update to the behavior of the agent interaction, specifically we combined the agent and bot views into one which is a significant enough change I would consider redoing this.

Reply:
What has visibly changed in the update since our recording? We did not see anything obvious.

Reply:
In prior versions of web chat, when you speak to an agent, that conversation takes place in a different panel and you can switch back and forth between the agent and the bot. That no longer happens. Now both conversations take place in the same panel with no ability to move back and forth.
![image](https://user-images.githubusercontent.com/25828402/218796014-3bf0f6fc-ed97-4aef-b441-ca0d881f154c.png)
![image](https://user-images.githubusercontent.com/25828402/218796205-3e03af11-b7dd-4d8d-a2e0-55abdee4b261.png)

In your image, at the end, when it returns to web chat from your widget, it briefly displays the old agent only panel. It is just a few seconds and not that easy to notice so perhaps it is not worth the effort to change. So I will leave it up to you guys to decide you want to do so.
![image](https://user-images.githubusercontent.com/25828402/218797191-0ecfed33-82e9-4511-a160-bfd3b795f65b.png)